pipelines/braindecode.py

```diff
@@ -0,0 +1,62 @@
+from braindecode.classifier import EEGClassifier
+from braindecode.models import ShallowFBCSPNet
+from sklearn.pipeline import Pipeline
+from skorch.callbacks import LRScheduler
+from skorch.dataset import ValidSplit
+from torch.cuda import is_available
+from torch.nn import NLLLoss
+from torch.optim import AdamW
+
+from moabb.pipelines.braindecode import (
+    BraindecodeClassifierModel,
+    CreateBraindecodeDataset,
+)
+
+
+# hard-coded for now
+n_classes = 2
+n_chans = 22
+input_window_samples = 1001
+# These values we found good for shallow network:
+lr = 0.0625 * 0.01
+weight_decay = 0
+batch_size = 64
+n_epochs = 4
+
+model = ShallowFBCSPNet(
+    n_chans,
+    n_classes,
+    input_window_samples=input_window_samples,
+    final_conv_length="auto",
+)
+device = "cuda" if is_available() else "cpu"
+
+clf = EEGClassifier(
```

Comment:
We need a scikit wrapper for that

Reply:
In fact, we don't need. Thank you for that!